Detectors/TPC/workflow/src/tpc-reco-workflow.cxx

```diff
@@ -157,6 +158,7 @@ WorkflowSpec defineDataProcessing(ConfigContext const& cfgc)
                                                 nLanes,                                            //
                                                 inputType,                                         //
                                                 cfgc.options().get<std::string>("output-type"),    //
+                                                cfgc.options().get<bool>("disable-root-input"),    //
```

Comment:
Apparently I had a copy/paste error on https://github.com/AliceO2Group/AliceO2/blob/375f0f6a3654ee74fc11729e0085a89619d53d2f/Detectors/TPC/workflow/src/tpc-reco-workflow.cxx#L170, could you please substitute the repeating line `o2::conf::ConfigurableParam::writeINI("o2tpcrecoflow_configuration.ini");` by 
````
// configure dpl timer to inject correct firstTFOrbit: start from the 1st orbit of TF containing 1st sampled orbit
  o2::raw::HBFUtilsInitializer hbfIni(configcontext, wf);
````
?